.env.example

```diff
@@ -1,6 +1,15 @@
+SERVER_URL=http://localhost:8080
```

Comment:
The application doesn't know which domain points at it. But it needs to know to which URL it needs to send users to for the signup email so I added it to the env. On production it will be `https://empire-and-foundation.herokuapp.com`